src/xcode/ENA/ENA/Source/Models/Exposure/ExposureManager.swift

```diff
@@ -246,17 +259,29 @@ final class ENAExposureManager: NSObject, ExposureManager {
 	}
 
 	private func changeEnabled(to status: Bool, completion: @escaping CompletionHandler) {
+		Log.info("Trying to change ENManager.setExposureNotificationEnabled to \(status).")
+
+		var hasChanged = false
 		manager.setExposureNotificationEnabled(status) { error in
 			if let error = error {
 				Log.error("Failed to change ENManager.setExposureNotificationEnabled to \(status): \(error.localizedDescription)", log: .api)
 				self.handleENError(error: error, completion: completion)
 				return
 			}
+			Log.error("Successfully changed ENManager.setExposureNotificationEnabled to \(status).", log: .api)
+			hasChanged = true
 			completion(nil)
 		}
+		
+		// Sometimes the ENF framework is broken. So we wait 1 seconds to ensure that the changed has been applied. Mostly, the ENF framework responds instantly, so we check after 2 seconds and show then an alert.
+		DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```

Comment:
The comment say 2 seconds but the code only 1? What is the correct value herer?

Reply:
the two is outdated. I'll fix that.